compiler/rustc_parse/src/parser/attr_wrapper.rs

```diff
@@ -513,18 +513,16 @@ fn make_attr_token_stream(
         }
     }
 
-    if break_last_token {
+    if break_last_token > 0 {
         let last_token = stack_top.inner.pop().unwrap();
         if let AttrTokenTree::Token(last_token, spacing) = last_token {
-            let unglued_first = last_token.kind.break_two_token_op().unwrap().0;
+            let unglued = last_token.kind.break_two_token_op(break_last_token).unwrap();
 
-            // An 'unglued' token is always two ASCII characters
```

Comment:
A useful comment removed (explaining why using `BytePos` arithmetic is ok).

Reply:
It was wrong as written (they're not always two chars) but I've reinstated it in a more correct form.